R/fit_SE.R

```diff
@@ -24,10 +26,8 @@ fit_SE <- function(se,
   checkmate::assert_number(pcutoff)
   checkmate::assert_number(cap)
   checkmate::assert_character(curve_type)
-  
-  req_assays <- c(averaged_assay)
-  lapply(req_assays, function(x) gDRutils::validate_se_assay_name(se, x))
-
+  checkmate::assert_true(all(curve_type %in% c("GR", "RV")))
```

Comment:
Or simpler?
```
checkmate::assert_character(curve_type, pattern = "GR|RV")
```

Reply:
In this case I think mine is easier to maintain over the suggested regular expression.

Reply:
.